summary_workflows/quantification/quantification_dockers/apaeval_consolidation/manage_assessment_data.py

```diff
@@ -8,8 +8,9 @@
 from argparse import ArgumentParser
 import numpy as np
 from assessment_chart import assessment_chart
+from assessment_chart import conf
 
-DEFAULT_eventMark = '2018-04-05'
+DEFAULT_eventMark = DEFAULT_eventMark()
```

Comment:
This assignment makes me feel uneasy as it feels very error-prone to name the variable exactly as the function. However, if you say you did it on purpose I can live with it.

Reply:
I find this weird too, so I just have it removed.

Reply:
Thank you! And well done removing the definition from the global scope, I hadn't realised it!

One last thing about this, do you think that `DEFAULT_eventMark` is a good name for a variable that currently holds a date? Honestly, I don't know what "eventMark" is supposed to mean, perhaps "event_date" would be more meaningful and the "DEFAULT" prefix can be removed definitely.

Reply:
Thanks for the suggestion! I have changed `DEFAULT_eventMark` to `event_date`. Also, the docker directories are now named `q2_<validation/consolidation/metrics>` instead of `apaeval_<validation/consolidation/metrics>`.